rules/ast/functions.go

```diff
@@ -291,6 +291,25 @@ func dropCommonLabelsImpl(timestamp clientmodel.Timestamp, args []Node) interfac
 	return vector
 }
 
+// === round(vector VectorNode) Vector ===
+func roundImpl(timestamp clientmodel.Timestamp, args []Node) interface{} {
+	// round to even
```

Comment:
How do you prefer corrections? as force pushes to the PR branch or as a new commit which you will squash before merging?